ios/build/Build/Intermediates.noindex/XCBuildData/PIFCache/project/PROJECT@v8_mod=1543348485_hash=d1b7560c4abbdd3aebce48be11a78dd2-json

```diff
@@ -0,0 +1 @@
9f177b5df9dad57e7ca4c8a0f24","path":"../node_modules/react-native-vector-icons/Fonts/MaterialCommunityIcons.ttf","sourceTree":"<group>","type":"file"},{"fileType":"file","guid":"d1b7560c4abbdd3aebce48be11a78dd2c537e24bfd19c248d958f63d8c56caa9","path":"../node_modules/react-native-vector-icons/Fonts/MaterialIcons.ttf","sourceTree":"<group>","type":"file"},{"fileType":"file","guid":"d1b7560c4abbdd3aebce48be11a78dd23e1013a23fd72fdae74b7d085fa710ca","path":"../node_modules/react-native-vector-icons/Fonts/Octicons.ttf","sourceTree":"<group>","type":"file"},{"fileType":"file","guid":"d1b7560c4abbdd3aebce48be11a78dd20664102268b4ab31e91d3be0bac269ea","path":"../node_modules/react-native-vector-icons/Fonts/SimpleLineIcons.ttf","sourceTree":"<group>","type":"file"},{"fileType":"file","guid":"d1b7560c4abbdd3aebce48be11a78dd2f2452297126207d8e60d98d10be6652b","path":"../node_modules/react-native-vector-icons/Fonts/Zocial.ttf","sourceTree":"<group>","type":"file"},{"fileTextEncoding":"iso-8859-2","fileType":"text.html","guid":"d1b7560c4abbdd3aebce48be11a78dd213cdc65b66223163b775eb6123594652","path":"../node_modules/react-native-youtube/assets/YTPlayerView-iframe-player.html","sourceTree":"<group>","type":"file"}],"guid":"d1b7560c4abbdd3aebce48be11a78dd2a6f3cb5000041ebcd3b437ba1a21e82c","name":"Resources","path":"","sourceTree":"<group>","type":"group"},{"children":[{"fileType":"wrapper.pb-project","guid":"d1b7560c4abbdd3aebce48be11a78dd24bd4dc3b7009de6bde6752bd5fa900a6","path":"../node_modules/react-native-vector-icons/RNVectorIcons.xcodeproj","sourceTree":"<group>","type":"file"},{"fileType":"wrapper.pb-project","guid":"d1b7560c4abbdd3aebce48be11a78dd2ee5d5c3e3205ba40086275f1707579c7","path":"../node_modules/react-native-youtube/RCTYouTube.xcodeproj","sourceTree":"<group>","type":"file"}],"guid":"d1b7560c4abbdd3aebce48be11a78dd2ccd5f8b06140721c3d34c18f22e41fbd","name":"Libraries","path":"","sourceTree":"<group>","type":"group"},{"children":[{"fileType":"archive.ar","guid":"d1b7560c4abbdd3aebce48be11a78dd2219c50c457895b11ca4bb6fa9612c795","path":"libRNVectorIcons.a","sourceTree":"<group>","type":"file"},{"fileType":"archive.ar","guid":"d1b7560c4abbdd3aebce48be11a78dd2b3ea056fbcedcf796021f0576dc98509","path":"libRNVectorIcons-tvOS.a","sourceTree":"<group>","type":"file"},{"fileType":"archive.ar","guid":"d1b7560c4abbdd3aebce48be11a78dd2b864a14bcc88b1668beebe3f150bd089","path":"libRCTYouTube.a","sourceTree":"<group>","type":"file"}],"guid":"d1b7560c4abbdd3aebce48be11a78dd2768d10628e746f80f2eae15080be2efb","name":"Recovered References","path":"","sourceTree":"<group>","type":"group"}],"guid":"d1b7560c4abbdd3aebce48be11a78dd2e73c1bdf006197cb41344814ac3b6201","name":"movement-vites","path":"","sourceTree":"<group>","type":"group"},"guid":"d1b7560c4abbdd3aebce48be11a78dd2","path":"/Users/Justin/Movement-Vitamins-App/ios/movement-vites.xcodeproj","projectDirectory":"/Users/Justin/Movement-Vitamins-App/ios","targets":["TARGET@v8_hash=8A7FE9388A27CE2830CF91823B64AE8F"]}
```

Comment:
we should .gitignore files like this if its not necessary that they are in the repo.

Reply:
removed